src/arrayMethodJoin.js

```diff
@@ -4,8 +4,30 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    const currentArrayLength = this.length;
+    const joinSeparator = String(separator);
```

Comment:
Do we really need this?
I guess result will be the same with and without `String()`
<img width="206" alt="image" src="https://github.com/mate-academy/js_array-method-join/assets/86789161/8a566390-c0db-41e2-95f2-6ca0e87f40b8">

Reply:
@AndrMar1939 reply to it